client/src/App.js

```diff
@@ -969,17 +976,39 @@ class App extends Component {
                   this.state.loggedIn ? <Redirect to={"/"}/> : 
                   <Signup
                       isAlumni={true}
-                      match={props}
+                      match={props.match}
+                      history={props.history}
+                  />
+              }
+          />
+          <Route exact path={`/register/alumni/:referrerId/:schoolId`} render={
```

Comment:
nit beyond scope of this PR: it would be nice to consolidate the logged in logic check
question within the scope of this PR: why does the register/alumni path have to have two block cases here (i.e. why use and exact match on the route in this case)? It looks like they are both taking in same props.